fighthealthinsurance/settings.py

```
@@ -750,25 +751,26 @@ def EXTERNAL_STORAGE_B(self):
                     minio = MinioStorage(
                         minio_client,
                         bucket_name=bucket,
                         auto_create_bucket=False,  # B2 issues
                     )
-                    print(f"Connected to minio at {self.MINIO_STORAGE_ENDPOINT}")
+                    logger.info(f"Connected to minio at {self.MINIO_STORAGE_ENDPOINT}")
 
                     return minio
                 else:
                     raise Exception("No storage endpoint configured")
         except Exception as e:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)